python/lsst/ts/hvac/mqtt_info_reader.py

```diff
@@ -0,0 +1,467 @@
+# This file is part of ts_hvac.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = [
+    "MqttInfoReader",
+    "data_dir",
+]
+
+import json
+import pandas
+import pathlib
+import re
+
+from lsst.ts.hvac.hvac_enums import (
+    CommandItem,
+    HvacTopic,
+    TelemetryItem,
+    TOPICS_ALWAYS_ENABLED,
+)
+
+# The names of the columns in the CSV file in the correct order.
+names = [
+    "floor",
+    "subsystem",
+    "variable",
+    "topic_and_item",
+    "publication",
+    "subscription",
+    "signal",
+    "rw",
+    "range",
+    "limits",
+    "unit",
+    "state",
+    "observations",
+    "notes",
+]
+
+# Find the data directory relative to the location of this file.
+data_dir = pathlib.Path(__file__).resolve().parents[3].joinpath("data")
```

Comment:
This looks like a constant, so please consider DATA_DIR.

Another option is to make `data_dir` an attribute of `MqttInfoReader`. If you do that in the constructor you can also check that the dir (and perhaps the others it reads) exists and raise a suitable error if not.

Also I gently suggest "/" instead of "joinpath" here and everywhere. It works on Windows and I personally feel it makes the code more readable.

Reply:
Done